mojave/items/storage/radiopack.dm

```diff
@@ -114,8 +130,22 @@
 	return
 
 /obj/item/radio/ms13/NCR/dropped(mob/user)
-	SHOULD_CALL_PARENT(0)
-	if(radio_pack)
-		radio_pack.attach_radio(user)
-	else
-		qdel(src)
+	. = ..()
+	if(user)
+		UnregisterSignal(user, COMSIG_MOVABLE_MOVED)
+	if(req_radio)
+		if(user)
+			to_chat(user, "<span class='notice'>The walkie talkie snaps back into place on the radiopack.</span>")
+		snap_back()
+
+/obj/item/radio/ms13/NCR/proc/snap_back()
+	if(!radiopack)
+		return
+	forceMove(radiopack)
+
+/obj/item/radio/ms13/NCR/doMove(atom/destination)
+	if(destination && (destination != radiopack.loc || !ismob(destination)))
+		if (loc != radiopack)
```

Comment:
This proc will probably need to be adjusted to not spam chat whenever you take it out or put it back in. Also not super pressing and will not delay a merge, but worth noting for future.

Reply:
He could just remove the to_chat if it's a problem.